### PR TITLE
Small tweaks to FAQ

### PR DESCRIPTION
**Pull request by shrayasr**
- There were unclosed `<p>` tags and unnecessary `<span>` tags lying around
- Merged 2 FAQs into one and made it simpler
- Small grammatical fixups

Request @harisibrahimkv to sign off on the text.

### CODE REVIEW

File: faq.html [hunk 1]
```diff
@@ -122,30 +122,26 @@ <h2 class="text-center">
...
-                    <span> Conference day(Oct 3rd and 4th): 08:00 - 17.30.</span>
+                  <span><b> Q. What are the workshop and conference timings like?</b><br></span>
+                  <span>A. Workshop day (Oct 2nd): 08:00 AM - 06:30 PM.<br/></span>
+                  <span> Conference day (Oct 3rd and 4th): 08:00 AM - 5.30 PM.</span>
                 </p><br>
                 <p>
-                   <span><b> Q. What all does Conference ticket(Oct 3&4) includes ? </b><br><span>
-                    <span> Full conference pass for two days . <br>Breakfast, Lunch and Hi-Tea .<br>It also include  PyCon India T-shirt which will be distributed  on Oct 4  </span>
-                </span>
+                  <span><b> Q. What all does Conference ticket(Oct 3&4) includes ? </b><br></span>
```
Comment (harisibrahimkv) [inline faq.html]: @shrayasr Make that "What all does the conference ticket (Oct 3&4) include?".

Comment (shrayasr) [inline faq.html]: Done. Sorry missed that one

**Comment by kracekumar** [comment]
@harisibrahimkv once you're ok, let me know.

**Comment by harisibrahimkv** [comment]
@kracekumar Go ahead.

**Comment by shrayasr** [comment]
Thanks :+1:

**Pull request merged**